eq-author/src/components/modals/QuestionnaireSelectModal/index.js

```diff
@@ -0,0 +1,111 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import styled from "styled-components";
+import { colors } from "constants/theme";
+import Modal from "components/modals/Modal";
+import Button from "components/buttons/Button";
+import ButtonGroup from "components/buttons/ButtonGroup";
+import ScrollPane from "components/ScrollPane";
+import Panel from "components/Panel";
+
+const ModalFooter = styled.div`
```

Comment:
```suggestion
const ModalFooter = styled.footer`
```

Would be good I think if we as a team start using HTML5 elements, as they're more semantic and some assistive technologies hook onto them when giving aid.

You can use `footer` as both a footer for the whole page, or as a footer for a section of content on the page.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/footer

Reply:
nice one - thanks Matt - makes good sense!